src/rgw/rgw_tools.cc

```diff
@@ -592,3 +594,27 @@ void rgw_tools_cleanup()
   delete ext_mime_map;
   ext_mime_map = nullptr;
 }
+
+void rgw_complete_aio_completion(librados::AioCompletion* c, int r) {
```

Comment:
it looks like this was mostly copied from CB_AioCompleteAndSafe in AioCompletionImpl.h. i worry that if librados changes, this copy could be missed and lead to bugs. could we use CB_AioCompleteAndSafe directly instead?
```c++
librados::CB_AioCompleteAndSafe cb(c->pc);
cb(r);
```